src/styles/blocks/catalog.scss

```diff
@@ -1,3 +1,5 @@
+@import "blocks/variable.scss";
```

Comment:
Yuo can import variables fite to main.scss and not to add it to two different files

Reply:
agree